### PR TITLE
Weekly sync 230424

### PR DESCRIPTION
**Pull request by wenchenvincent**
https://github.com/ROCmSoftwarePlatform/frameworks-internal/issues/4263

### CODE REVIEW

File: tensorflow/compiler/xla/service/gpu/tests/BUILD [hunk 1]
```diff
@@ -812,10 +812,6 @@ xla_cc_binary(
     ]) + if_rocm_is_configured([
         "//tensorflow/tsl/platform:rocm_rocdl_path",
         "//tensorflow/compiler/xla/service/gpu:amdgpu_compiler_impl",
-<<<<<<< HEAD
-	"//tensorflow/tsl/platform:rocm_rocdl_path",
```
Comment (jayfurmanek) [inline tensorflow/compiler/xla/service/gpu/tests/BUILD:813]: This was added in: https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/commit/dfb7dd0b2f7e6c6bbcd3f1dbc9e299be9554b4a7

At the time, it was needed. Curious if that is still the case.

**Comment by jayfurmanek** [comment]
It looks like we need this one now too:
https://github.com/tensorflow/tensorflow/commit/e6aa56fad3d7d24561d9e7321a4b6b991b74762f

Since estimator got ahead of us with:
https://github.com/tensorflow/estimator/commit/ad72355f9f766fda7bf5cdedd9146b8d787458db

**Comment by wenchenvincent** [comment]
> It looks like we need this one now too: [tensorflow@e6aa56f](https://github.com/tensorflow/tensorflow/commit/e6aa56fad3d7d24561d9e7321a4b6b991b74762f)
> 
> Since estimator got ahead of us with: [tensorflow/estimator@ad72355](https://github.com/tensorflow/estimator/commit/ad72355f9f766fda7bf5cdedd9146b8d787458db)

Is this commit to fix the following error?
AttributeError: module 'tensorflow.python.ops.ref_variable' has no attribute '_to_proto_fn'

Now all the CI test failures are due to this error. But when I tried to run them on a local MI60, there were no failures. So it seems that there is some difference between the docker container used by the CI and rocm/tensorflow-autobuilds:latest that led to the discrepancy.

**Comment by jayfurmanek** [comment]
> Is this commit to fix the following error? AttributeError: module 'tensorflow.python.ops.ref_variable' has no attribute '_to_proto_fn'

yes

> Now all the CI test failures are due to this error. But when I tried to run them on a local MI60, there were no failures. So it >seems that there is some difference between the docker container used by the CI and rocm/tensorflow-autobuilds:latest that >led to the discrepancy.

yeah, the first commit above went in to TF last week. The second one followed in TFE yesterday and wound up in `tf-estimator-nightly` May 01 build. The "Ubuntu-GPU-*" jobs build the old container each time and pull in the latest `tf-estimator-nightly` each time which is why were are seeing it there. The "gpu-non-pip' jobs pull and use the tensorflow-build container which get published on each new push to the repo, so those don't yet have the tf-estimator-nightly May 01 build.

Every once in a while there are some interdependencies like this and we got bit here since this sync is a little late. I just did a test build locally (with tf-estimator-ngihtly May01) and this commit does indeed fix the issue. I'll push the cherry-pick.

**Review by jayfurmanek — APPROVED** [review]

**Pull request merged**